tests/fpga/streaming_memory_test.py

```diff
@@ -237,6 +313,380 @@ def test_streaming_and_composition():
     return sdfg
 
 
+@xilinx_test()
```

Comment:
What happens if you test this (and the following) also with intel_fpga?

Reply:
Timed out: [vec_add, mat_add, tensor_add, multistream, multistream_with_deps, map_order]
Compiler failure: [mapnests]
Pass: [vec_add_1, vec_add_1_symbolic, mat_add_symbol]